pkg/common/helper.go

```diff
@@ -230,11 +230,11 @@ func retryUntilSuccessful(duration time.Duration,
 	if lastErr != nil {
 
 		// wrap last error
-		return errors.Wrapf(lastErr, timedOutErrorMessage)
+		return errors.Wrapf(lastErr, "%s", timedOutErrorMessage)
 	}
 
 	// duration expired without any last error
-	return errors.Errorf(timedOutErrorMessage)
+	return errors.Errorf("%s", timedOutErrorMessage)
```

Comment:
what is the difference?

Reply:
It was needed due to this [govet lint](https://github.com/golangci/govet/blob/master/print.go#L303-L364) error:
```
printf: non-constant format string in call to github.com/nuclio/errors.Errorf (govet)
        return errors.Errorf(timedOutErrorMessage)
```

Baiscally when using `Printf` (as `Errorf` uses in this case) you should use format strings for safer usage and escaping